src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2357,7 +2357,6 @@ internal void Run(bool inputLoopIsNested)
                 Dbg.Assert(ui != null, "Host.UI should return an instance.");
```

Comment:
@vexx32, test results were not published for PowerShell-CI-windows at vstfs:///Build/Build/26672